Week3/homework/step2-3.js

```diff
@@ -19,8 +23,10 @@ function repeatStringNumTimesWithWhile(str, num) {
   let result = '';
 
   // Replace this comment and the next line with your code
-  console.log(str, num, result);
-
+  while (num > 0) {
+    result += str;
+    num--;
+  }
```

Comment:
Ah. You missed adding the logic of checking `num` like you did in `for` loop here.